console/console-init/ui/src/components/common/index.ts

```diff
@@ -7,7 +7,7 @@ export * from "./About";
 export * from "./AddressSpaceListFormatter";
 export * from "./ConnectionListFormatter";
 export * from "./DialogPrompt";
-export * from "./EmptyLinks";
+export * from "../../modules/connection-detail/components/EmptyConnectionLinks/EmptyConnectionLinks";
```

Comment:
we can remove this export from here

Reply:
Agreed, this got here because of the auto path update in VS. I saw this after sending the PR.